### ISSUE TITLE
making e-speak speak Turkish instead of English in languges other than Latin

### ISSUE THREAD

**Issue by fatihyuksek1** [espeak-ng/espeak-ng#1222]
I want to make e-speak speak Turkish instead of English for languages which use non latin characters in NVDA.
How do I do that?

**Comment by Qchristensen** [comment]
There are several ways of changing languages in NVDA.  By default it will change automatically when it encounters text which declares a new language (for instance using the "lang" attribute on a web page).

You can create configuration profiles in NVDA set to different languages keystrokes to change between them.  (If you need to learn how to do that, then please refer to the Basic Training for NVDA module from https://www.nvaccess.org/shop ).

I don't switch languages often, but I think NVDA uses the default language you have set if it doesn't have another instruction.

**Comment by sukiletxe** [comment]
There was an option in eSpeak which let you specify an alternative alphabet in a voice file, like this:
```
alphabet2  cyr  ru  
```

Sadly this is missing from Espeak NG.

**Comment by jaacoppi** [comment]
Duplicate of #999
